nHentai_downloader/nHentai_downloader.user.js

```diff
@@ -21,6 +21,8 @@
 jQuery(function ($) {
     'use strict';
 
+    var extractedInfo = null;
```

Comment:
What is task of `extractedInfo`? If it is used to write manga information, you will have to edit script every time download.